Library/Formula/vdirsyncer.rb

```diff
@@ -12,13 +12,13 @@ class Vdirsyncer < Formula
     sha256 "d0ad5bd159acdfeaf28aec0abda9f34d35750c4377624a365180c24798a8059c" => :mavericks
   end
 
-  option "without-keyring", "Build without python-keyring support"
+  option "with-remotestorage", "Build with support for remote-storage"
```

Comment:
It is an optional dependency for using webdav as storage. If you don't use it, the dep isn't needed, hence I made it optional (just like the keyring option in the versions before).